webapp/components/Editor/Editor.vue

```diff
@@ -200,12 +200,173 @@ export default {
     EditorMenuBubble,
   },
   props: {
-    users: { type: Array, default: () => [] },
-    hashtags: { type: Array, default: () => [] },
+    users: { type: Array, default: () => null }, // If 'null', than the Mention extention is not assigned.
+    hashtags: { type: Array, default: () => null }, // If 'null', than the Hashtag extention is not assigned.
     value: { type: String, default: '' },
     doc: { type: Object, default: () => {} },
   },
   data() {
+    // Set array of optional extensions by analysing the props.
+    let optionalExtensions = []
+    // Don't change the following line. The functionallity is in danger!
+    if (this.users) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-15T22:55:41Z" title="Friday, August 16th 2019, 12:55:41 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -204,12 +204,173 @@ export default {
     EditorMenuBubble,
   },
   props: {
-    users: { type: Array, default: () => [] },
-    hashtags: { type: Array, default: () => [] },
+    users: { type: Array, default: () => null }, // If 'null', than the Mention extention is not assigned.
+    hashtags: { type: Array, default: () => null }, // If 'null', than the Hashtag extention is not assigned.
     value: { type: String, default: '' },
     doc: { type: Object, default: () => {} },
   },
   data() {
+    // Set array of optional extensions by analysing the props.
+    let optionalExtensions = []
+    // Don't change the following line. The functionallity is in danger!
+    if (this.users) {
```


Can you explain in which situations we would render `<hc-editor>` without users or hashtags?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-16T07:16:36Z" title="Friday, August 16th 2019, 9:16:36 am +02:00">Aug 16, 2019</time>_

---


Yes, for sure:
In the comments hashtags are not used.

In the future:
If you write a description of yourself in your user profile hashtags and mentions should not be used.